Core/Plugins/PersistentGlobalStringCache.cs

```diff
@@ -126,13 +126,13 @@ public IEnumerable<IIssue> GetIssues()
     /// <summary>
     /// Not for you. Don't use this. It creates a separate file for every key. Wraps a singleton
     /// </summary>
-    public class PeristentGlobalStringCache : IPersistentStringCache, IIssueProvider
+    public class PersistentGlobalStringCache : IPersistentStringCache, IIssueProvider
```

Comment:
API Change

Reply:
I think this will be OK. It's not likely to be used.